ProjectFiles/Assets/Scripts/Game/Entity/ClientWorld.cs

```diff
@@ -0,0 +1,44 @@
+using System.Collections.Generic;
+using Network;
+using UnityEngine;
+
+namespace Game.Entity {
+    public class ClientWorld : World {
+
+        /// <summary>
+        /// For use in standalone mode. 
+        /// </summary>
+        public void SpawnPlayers() {
+            var spawnPos = new PlayerPosition {
+                Pos = new Vector3 {
+                    x = -6.82f,
+                    y = 2.19f,
+                    z = -0.7f,
+                },
+                Rot = new Quaternion(),
+            };
+            foreach (var kv in Players) {
+                kv.Value.Spawn(spawnPos);
+            }
+        }
```

Comment:
This could very easily go wrong - someone would use this method by mistake and be suprised that everyone has spawned in the same place. I'd change the method signature to say `standalone` or maybe wrap it up in on of those `IF` macros to say that this code only ever gets compiled for the standalone mode?